chore: update firmware changelogs

obrusvit opened the pull request:
[no changelog]

<!--
For core developers:
- Assign yourself to the PR.
- Set the priority to match the original issue.
- Add the PR to the current sprint.
- If it's a draft PR, mark it as "In Progress."
- If it's a final PR, mark it as "Needs Review."

For external contributors:
- Please open an issue before submitting a PR so we can discuss whether we want to proceed with it.
-->

mmilata commented on core/CHANGELOG.md:
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
+## [2.9.3] (21st October 2025)
+
+### Added
+- Support using both Tropic and Optiga to protect PIN.  [#5845]
```
This should probably be `[T3W1]`.

Also maybe a good time to start CHANGELOG.T3W1.md.

mmilata commented on core/CHANGELOG.md:
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
+### Fixed
+- [T3W1] Cache THP host info also during credential-based pairing.  [#5867]
+- Fix incorrect chunkified address rendering.  [#5882]
+- Exit pairing screen when already paired host connects.  [#5897]
```
Also.

obrusvit committed:
chore: changelog adjustements

- fix some missing models in CHANGELOG.md
- add T3W1 changelog
- make some changelogs T3W1-specific
- modify changelog.py to allow for keeping the model-specific changelog
up to a minimal version

[no changelog]

obrusvit committed:
chore: update firmware changelogs

[no changelog]
